rootfs-builder/README.md

```diff
@@ -124,3 +85,88 @@ must be met:
    $ # build image based rootfs created above
    $ ./image-builder/image_builder.sh "${PWD}/fedora_rootfs"
    ```
+
+## Adding support for a new guest OS
+
+The `rootfs.sh` script will check for immediate sub-directories
+containing the following expected files:
+
+- A `bash(1)` script called `rootfs_lib.sh`
+
+  This file must contain a function called `build_rootfs()` which
+  must receive as its first argument the path to where the rootfs will be
+  created.
+
+  Path: `rootfs-builder/<distro>/rootfs_lib.sh`.
+
+
+- A `bash(1)` script called `config.sh`
+
+  This represents the specific configuration for `<distro>`. It must
+  provide configuration specific variables for the user to modify as needed.
+  The `config.sh` file will be loaded before executing `build_rootfs()` to
+  provide all the needed configuration to the function.
+
+  Path: `rootfs-builder/<distro>/config.sh`.
+
+### Create template files
+
+To create a directory with the expected file structure run:
+
+```
+$ make -f template/Makefile  ROOTFS_BASE_NAME=my_new_awesome_rootfs
+```
+
+After running the command above, a new directory will be created in
+`rootfs-builder/my_new_awesome_rootfs/`. To verify it is one of the
```

Comment:
Lines 121-123 rewrite (I'm not completely sure what this sentence is trying to relay, so I might have changed meaning here): To verify the directory can be used to build a rootfs, run `./rootfs.sh -h`. Running this script shows 'my_new_awesome_rootfs` as one of the options for use.

Reply:
That looks good to me. I've removed the duplication of showing the command below the text though and and going to add a reference to the section "Creating a rootfs" as that is where the command is ultimately specified.